pkg/controller/shared_state.go

```diff
@@ -1536,18 +1537,24 @@ func (c *SharedState) createAzVolume(ctx context.Context, desiredAzVolume *azdis
 
 	var err error
 	var azVolume *azdiskv1beta2.AzVolume
+	var originalAzVolume *azdiskv1beta2.AzVolume
```

Comment:
`originalAzVolume` is just a copy of  `azVolume`. We can remove its usage.

Reply:
`azVolume` is updated in line 1587: `azVolume.Status.Annotations = azureutils.AddToMap(azVolume.Status.Annotations, statusAnnotation...)` without updating our clients. I feel like we need `originalAzVolume` to track the unchanged object, otherwise we may get `conflict` error when we update CRI. Please let me know if my understanding is wrong.

Reply:
This is a pre-existing bug. See my comment below.